pages/classes/cunning.md

```diff
@@ -6,11 +6,29 @@
 
 | Level | Proficiency | Class Features  |
 | ----  | ----------- |- |
-| 1st   | +2          | Archetype, Proficient: [Cunning](pages/characters/attributes.md?id=cunning) [saves](pages/rules/rolling.md?id=saves), Proficient: Any 1 Skill |
+| 1st   | +2          | Cunning Attack, Archetype, Proficient: [Cunning](pages/characters/attributes.md?id=cunning) [saves](pages/rules/rolling.md?id=saves), Proficient: Any 1 Skill |
+
+### Cunning Attack
+
+You can deal an extra 1d6 damage to one creature you hit with an attack if you have advantage.
 
 ### Archetype
 
-Choose an archetype and gain it's features. The Cunning archetypes include: **Rogue**.
+Choose an archetype and gain it's features. The Cunning archetypes include: Archer, and Rogue.
+
+## Archer
+
+You are a skilled bowman, trained in bow and arrow. With a keen eye, you rarely miss your mark.
+
+| Level | Archetype Features |
+| ----  | ------------------ |
+| 1st   | Archery, Aim, Proficient: Perception            |
+
+### Aim
+
+You can use a bonus action to line-up your shot on a particular target. Choose a creature you can see within range and mark it. For 1 minute you don't need advantage on your **cunning attack** for ranged attacks against the marked target, if you don't have disadvantage on your attack.
```

Comment:
Flawed: basically means you can use your cunning attack whenever you want.

This leads onto a discussion: in 5E sneak attack is generally supposed to be applied to most attack rolls. Sort of makes it a bit meaningless... just give cunning attack to all attacks?